scripts/gcp_common

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+readonly IMAGE_NAME='oak-gcp-hello-world'
+readonly DISK_NAME='oak-hello-world-disk'
+readonly INSTANCE_NAME='oak-hello-world-instance'
+readonly NETWORK_TAG='oak-hello-world-network'
+readonly FIREWALL_RULE='oak-hello-world-access'
+readonly ZONE='europe-west2-c'
+
+if [[ -z "${GCP_PROJECT_ID:+''}" ]]; then
```

Comment:
Just curious, why is the `:+''` part needed?

Reply:
Doesn't that mean that this check is superfluous then?

Reply:
This just gives a better error message, making it clear that it is expecting a non-empty environment variable to be set. Without these checks the error is something like 'Unbound variable GCP_PROJECT_ID'.